LoopKit/HealthStoreUnitCache.swift

```diff
@@ -52,6 +56,14 @@ public class HealthStoreUnitCache {
             return unit
         }
 
+        guard let unit = getHealthStoreUnitAndUpdateCache(for: quantityTypeIdentifier) else {
```

Comment:
[nit] Just `return getHealthStoreUnitAndUpdateCache(for: quantityTypeIdentifier)`?